docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -107,7 +107,14 @@ To classify an image:
         dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
     ```
 
-4. Configure `g_engine_config` with classification model data to classify image. In the following example, TensorFlow Lite model is used and `data.tflite` and `label.txt` are in `<OwnDataPath>`. Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models):
+4. Configure `g_engine_config` with classification model data to classify image.
+   The default engine is configured by the system. You can see the supported engines in Media Vision Inference API reference (for
+		   [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f)
+		   and
+		   [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f).
```

Comment:
Please see the comment for the line 111.

As that comment include the line 112~114, please remove line 112~114.